docs/pages/build-reference/e2e-tests.mdx

```diff
@@ -366,39 +459,27 @@ The full example from this guide is available at https://github.com/expo/eas-tes
 
 ### Using development builds to speed up test run time
 
+> **Warning** This might not work properly on Android.
```

Comment:
I haven't been able to run this on Android but I haven't put too much effort into it. We can research this after this PR is merged. @douglowder could you please look into this?

Reply:
I will look into this for sure 👍

Reply:
This does not work on Android -- the issue is that Detox has never made `device.openUrl()` work on Android. I'll take an action item to see if there is a way of getting this to work by executing adb commands.

Reply:
@douglowder - can you create a linear task to track that?

Reply:
@brentvatne https://linear.app/expo/issue/ENG-6679/make-eas-tests-example-work-with-android-debug-build-and-detox